packages/valory/skills/decision_maker_abci/behaviours/base.py

```diff
@@ -216,45 +266,6 @@ def check_balance(self) -> WaitableConditionType:
         self.context.logger.info(f"The safe has {native} xDAI and {collateral}.")
         return True
 
-    def _calculate_kelly_bet_amount(
```

Comment:
Antipattern to have this as an instance method. It's much harder to test this way